content/3_Silver/Graph_Traversal.mdx

```diff
@@ -1082,6 +1082,47 @@ public class BuildingTeams {
 ```
 
 </JavaSection>
+<PySection>
+
+Note that the below solution passes in CPython3 but exceeds the time limit on test case #7 in PyPy3.
```

Comment:
wrap in warning and just say you have to submit w/ cpython

Reply:
Got it!